visualisation/mean_spectra.py

```diff
@@ -0,0 +1,88 @@
+import pandas as pd
+import peakutils
+import plotly.graph_objects as go
+import streamlit as st
+
+from processing import save_read
+from processing import utils
+from . import draw
+
+MS = "Mean spectrum"
+AV = "Average"
+BS = "Baseline"
+RS = "Raman Shift"
+FLAT = "Flattened"
+COR = "Corrected"
+ORG = "Original spectrum"
+RAW = "Raw Data"
+OPT = "Optimised Data"
+NORM = "Normalized"
+
+
+def show_mean_plot(df, plots_color, template, spectra_conversion_type):
+    global col1
```

Comment:
Nie :no_entry: 
nigdy :no_entry_sign: 
przenigdy :no_good: 
Jak kiedyś będziesz miał ochotę napisać `global` w pytonie to wyłącz komputer.
Takie rzeczy tylko do debugowania albo jak robisz gigantyczny system i już totalnie musisz coś odjebać, ale to w osateczności. 

Tutaj tak samo jak wyżej musimy rozbić funkcjonalność na  przetwarzanie danych i na ich wyświetlanie.
1. Napierw rysujemy suwaki i zczytujemy z nich dane (wywołanie takiej/takich funkcji w `vis.py`). 
To powinno być super proste, więc suwaki możesz zrobić bezpośrednio w `vis.py`
2. potem wywołujemy funkcję, która na tej podstawie przetwarza dane i je zwraca
3. potem wywołujemy funkcję, która robi z tego plot'a
4. potem w vis.py robimy `st.write(fig)`

Reply:
okay, tak jak wyżej. mozemy to zrobic spotykajac sie albo na pycharm stream